src/api/network/asio/Imp.cpp

```diff
@@ -377,26 +355,22 @@ auto Asio::Imp::Receive(
         bufData.second,
         [this, connection{space(id)}, type, bufData, address{endpoint.str()}](
             const auto& e, auto size) {
-            data_socket_->Send([&] {
-                const auto& [index, buffer] = bufData;
-                auto work =
-                    opentxs::network::zeromq::tagged_reply_to_connection(
-                        reader(connection),
-                        e ? value(WorkType::AsioDisconnect) : type);
-
-                if (e) {
-                    LogVerbose()(OT_PRETTY_CLASS())("asio receive error: ")(
-                        e.message())
-                        .Flush();
-                    work.AddFrame(address);
-                } else {
-                    work.AddFrame(buffer.data(), buffer.size());
-                }
-
-                OT_ASSERT(1 < work.Body().size());
-
-                return work;
-            }());
+            const auto& [index, buffer] = bufData;
+            auto work = opentxs::network::zeromq::tagged_reply_to_connection(
+                reader(connection), e ? value(WorkType::AsioDisconnect) : type);
+
+            if (e) {
+                LogVerbose()(OT_PRETTY_CLASS())("asio receive error: ")(
+                    e.message())
+                    .Flush();
+                work.AddFrame(address);
+            } else {
+                work.AddFrame(buffer.data(), buffer.size());
+            }
+
+            OT_ASSERT(1 < work.Body().size());
```

Comment:
Do we want to keep asserts in production code ? Or they are disabled in release build?

Reply:
this is just log

Reply:
i mean this : OT_ASSERT(1 < work.Body().size());